src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
```

Comment:
- [x] -[Optional]
kindly add the language to the HTML to ensure that you follow the HTML best practice

Reply:
Thank You @Ouail-01 , I will make the changes and create request a new review